sympy/physics/vector/frame.py

```diff
@@ -621,13 +621,16 @@ def _rot(axis, angle):
                 from sympy.physics.vector.functions import kinematic_equations
                 q1, q2, q3 = amounts
                 u1, u2, u3 = symbols('u1, u2, u3', cls=Dummy)
-                templist = kinematic_equations([u1, u2, u3], [q1, q2, q3],
-                                               rot_type, rot_order)
-                templist = [expand(i) for i in templist]
-                td = solve(templist, [u1, u2, u3])
-                u1 = expand(td[u1])
-                u2 = expand(td[u2])
-                u3 = expand(td[u3])
+                if all(i.is_number for i in [q1, q2, q3]):
```

Comment:
I'm not sure what is_number does. I think it should return zeros if all of the u's are not functions of dynamicsymbols._t.